nav2_bt_navigator/src/navigate_to_pose_behavior_tree.cpp

```diff
@@ -30,6 +35,12 @@ NavigateToPoseBehaviorTree::NavigateToPoseBehaviorTree(rclcpp::Node::SharedPtr n
   // Register our custom action nodes so that they can be included in XML description
   factory_.registerNodeType<nav2_tasks::ComputePathToPoseAction>("ComputePathToPose");
   factory_.registerNodeType<nav2_tasks::FollowPathAction>("FollowPath");
+  factory_.registerNodeType<nav2_tasks::StopAction>("Stop");
+  factory_.registerNodeType<nav2_tasks::BackUpAction>("BackUp");
+  factory_.registerNodeType<nav2_tasks::SpinAction>("Spin");
+
```

Comment:
When you rebase, make sure that we're not registering the Parallel node. This is now done by the BT library (he's fixed this issue).

Reply:
OK